ddsp/training/ddsp_run_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020 The DDSP Authors.
```

Comment:
this should be `ddsp/training/cloud_test.py`

Reply:
Done.